### ISSUE TITLE
Docs need updating to reflect api key requirements in ^3.0.0-beta.23

### ISSUE THREAD

**Issue by gregbayer** [weaviate/typescript-client#137]
The V3 docs say the api key should be passed as a string when connecting in custom mode. See here:

https://weaviate.io/developers/weaviate/client-libraries/typescript/typescript-v3

```import weaviate from 'weaviate-client'

const client = await weaviate.client({
    rest: {
      host: 'WEAVIATE_INSTANCE_HOST_NAME',
      port: 8080,
      secure: true
    },
    grpc: {
      host: 'WEAVIATE_INSTANCE_HOST_NAME',
      port: 50051,
      secure: true
    },
    auth: {
      apiKey: process.env.WEAVIATE_API_KEY || ''
    },
    headers: {
      'X-OpenAI-Api-Key': process.env.OPENAI_API_KEY || ''
    }
  }
)
 
console.log(client)
```

However, I was only able to get the client to connect when wrapping the api key string like this. Otherwise the client validation code drops the key.

```
auth: {
  apiKey: new weaviate.ApiKey(process.env.WEAVIATE_API_KEY || '')
}
```

**Comment by tsmith023** [comment]
Hi @gregbayer, thanks for calling this one out! I think this is a regression so I'll add back support for detecting a string as an API key in the next RC release

**Comment by tsmith023** [comment]
Fixed in `v3.0.0-rc.3`
